src/mock-doc/test/mediaMatch.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { MockWindow } from '../window';
```

Comment:
Minor - to conform with the file nomenclature of this directory, can you rename this file for me?
```
mv src/mock-doc/test/matchMedia.spec.ts src/mock-doc/test/match-media.spec.ts
```

Reply:
Done making requested changes.